docs/docs/getting-started/attestation-crafting.md

```diff
@@ -64,19 +64,19 @@ To create an attestation two things are required, the Chainloop crafting tool an
 
 The crafting tool is currently bundled within Chainloop command line tool. To install it just follow the [installation](installation) instructions.
 
-The robot account was created during the [previous step](./workflow-definition#robot-account-creation) and it's required during all the stages of the crafting process. It can be provided via the `--token` flag or the `$CHAINLOOP_ROBOT_ACCOUNT` environment variable.
+The robot account was created during the [previous step](./workflow-definition#robot-account-creation) and it's required during all the stages of the crafting process. It can be provided via the `--token` flag or the `$CHAINLOOP_TOKEN` environment variable.
```

Comment:
there is another reference in the docs to the old CHAINLOOP_API_TOKEN that can be found here https://docs.chainloop.dev/reference/operator/api-tokens, we need to change it there too